sdk/nodejs/common/errors/DaggerSDKError.ts

```diff
@@ -0,0 +1,46 @@
+import { log } from "../utils.js"
+
+export interface DaggerSDKErrorOptions {
+  cause?: Error
+}
+
+/**
+ * The base error. Every other error inherits this error.
+ * @see [DaggerSDKError - Dagger.io](https://docs.dagger.io/current/sdk/nodejs/reference/classes/common_errors.DaggerSDKError)
```

Comment:
The `@see` param links the page to itself.
eg: https://deploy-preview-3947--devel-docs-dagger-io.netlify.app/current/sdk/nodejs/reference/classes/common_errors.DaggerSDKError

I guess we should remove it.